test/e2e/e2e_test.go

```diff
@@ -1129,32 +1045,7 @@ func TestE2E(t *testing.T) {
 	t.Run("serving-cert-annotation", func(t *testing.T) {
```

Comment:
could you add a comment explaining that this situation is only tmp.

Reply:
Updated